### PR TITLE
RPL_BANLIST: Document optional parameters

### LINKED ISSUE #124: RPL_BANLIST format differs in reality
According to tommy, they saw this format in use on libera:
```
367 <client> <chan> <mask> <banned_by> <banned_at>
```

The [defs list](https://defs.ircdocs.horse/defs/numerics.html#rpl-banlist-367 contains a different definition, so it makes sense to check out different ircds and see what's in more use and maybe mark the one currently on the defs list as deprecated and mark ^ as the current widely-accepted format for this numeric.

### PR DESCRIPTION
**Pull request by tommyrot**
resolves #124

### CODE REVIEW

**Comment by progval** [comment]
Looks good

File: _includes/modern-appendix.md [hunk 1]
```diff
@@ -562,10 +562,12 @@ Sent as a reply to the [`NAMES`](#names-message) command, this numeric specifies
 
 ### `RPL_BANLIST (367)`
 
-      "<client> <channel> <mask>"
+      "<client> <channel> <mask> [<who> [<set-ts>]]"
 
 Sent as a reply to the [`MODE`](#mode-message) command, when clients are viewing the current entries on a channel's [ban list](#ban-channel-mode). `<mask>` is the given mask on the ban list.
 
+`<who>` and `<set-ts>` are optional and MAY be included in responses. `<who>` is either the nickmask of the client that set the ban or the server name, and `<set-ts>` is the unix timestamp of when the ban was set.
```
Comment (SadieCat) [inline _includes/modern-appendix.md:569]:
```suggestion
`<who>` and `<set-ts>` are optional and MAY be included in responses. `<who>` is either the nickmask of the client that set the ban or the server name, and `<set-ts>` is the UNIX timestamp of when the ban was set.
```

File: _includes/modern-appendix.md [hunk 2]
```diff
@@ -562,10 +562,12 @@ Sent as a reply to the [`NAMES`](#names-message) command, this numeric specifies
 
 ### `RPL_BANLIST (367)`
 
-      "<client> <channel> <mask>"
+      "<client> <channel> <mask> [<who> [<set-ts>]]"
 
 Sent as a reply to the [`MODE`](#mode-message) command, when clients are viewing the current entries on a channel's [ban list](#ban-channel-mode). `<mask>` is the given mask on the ban list.
 
+`<who>` and `<set-ts>` are optional and MAY be included in responses. `<who>` is either the nickmask of the client that set the ban or the server name, and `<set-ts>` is the UNIX timestamp of when the ban was set.
```
Comment (progval) [inline _includes/modern-appendix.md:569]: Insp and Unreal send only a nick as `<who>`, not the whole nickmask

**Comment by progval** [comment]
I don't see any impl that provides `<who>` without `<set-ts>`. So you can probably remove the inner pair of brackets

**Comment by progval** [comment]
Could you merge master with you branch? There is a conflict on ``[`MODE`](#mode-message)``

**Pull request closed without merging**